nan_persistent_12_inl.h

```diff
@@ -81,6 +87,8 @@ class Global : public v8::Global<T> {
     v8::PersistentBase<T>::Reset(v8::Isolate::GetCurrent(), other);
   }
 
+  Global Pass() { return static_cast<Global&&>(*this); }  // NOLINT(build/c++11)
```

Comment:
This implementation of `Global` is conditionally compiled only for V8 4.3 and up.